.circleci/config.yml

```diff
@@ -58,6 +58,9 @@ jobs:
           command: bash <(curl -s https://codecov.io/bash)
           # Ref: https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute
           when: always
+      - run:
```

Comment:
How does this work? I see a "run" block already below. Can we have 2 blocks with the same tag?

Reply:
Yes, circleCI can accept a list of steps(or maybe runs) on a workflow

Reply:
ok